Samples/RenderGraphViewer/RenderGraphViewer.cpp

```diff
@@ -55,6 +55,36 @@ void RenderGraphViewer::onGuiRender(SampleCallbacks* pSample, Gui* pGui)
         }
     }
 
+    // allows for editing of lights while viewing graph output
+    if (mpScene && pGui->beginGroup("Scene Settings"))
```

Comment:
Move this to `SceneRenderer::renderUI()` and open an issue saying we need to finish the implementation.